jwql/instrument_monitors/common_monitors/dark_monitor.py

```diff
@@ -522,6 +522,9 @@ def process(self, file_list):
         mean_slope_file = self.save_mean_slope_image(slope_image, stdev_image, slope_files)
         logging.info('\tSigma-clipped mean of the slope images saved to: {}'.format(mean_slope_file))
 
+        # Free up memory
+        del slope_image_stack
```

Comment:
Nice, out of curiosity, how much memory does this free up?

Reply:
If depends on how many exposures were used in the dark current calculations. This deletes the stack of input images from which a mean image is constructed.

Reply:
Cool thanks! Merging.